.vib/spring-cloud-dataflow/cypress/cypress/integration/spring_cloud_dataflow_spec.js

```diff
@@ -0,0 +1,115 @@
+/// <reference types="cypress" />
+import { random } from './utils.js';
+import { importAnApplication, createATask } from './prepare_app_state.js';
+
+it('allows getting Spring Cloud Dataflow info', () => {
+  cy.visit('/dashboard');
+  cy.get('.signpost-trigger').click();
+  cy.get('.signpost-content-body')
+    .invoke('text')
+    .should('match', /.*[0-9].*/);
+  cy.contains('button', 'More info').click();
+  cy.contains('.modal-content', 'Core')
+    .and('contain', 'Dashboard')
+    .and('contain', 'Shell');
+});
+
+it('allows a stream to be created and deployed', () => {
+  cy.fixture('applications').then((application) => {
+    importAnApplication(application.newApplication.streamApplicationType);
+  });
+  cy.visit('dashboard/#/streams/list');
+  cy.contains('button', 'Create stream(s)').click();
+  cy.fixture('streams').then((stream) => {
+    cy.get('.CodeMirror-line').type(stream.newStream.type);
+  });
+
+  cy.contains('#v-2', 'mongodb').and('contain', 'cassandra');
```

Comment:
This is directly related to the test data. I think that changing the stream type in the fixture will make the test fail.

Reply:
I moved this to the application data fixture, hope it's better now!